packages/SystemsThinkingWorkbench-Core.package/STWSimulation.class/instance/doubleClick..st

```diff
@@ -0,0 +1,6 @@
+interaction
+doubleClick: evt
+	"create a new node"
+
+	self addNodeAt: evt cursorPoint.
+	self firstClicked: nil.
```

Comment:
what do you mean by: still need? we need to reset the first clicked node, because we said, that we reset any connection creation when we create a node